src/main/java/entities/User.java

```diff
@@ -241,53 +201,51 @@ public List<Timer> getTimers() {
      * Adds an invitation to this User's outgoingInvitations
      * @param invitation The new invitation to add
      */
-//    public void addOutgoingInvitation(Invitation invitation) {
-//        this.outgoingInvitations.add(invitation);
-//    }
+    public void addOutgoingInvitation(Invitation invitation) {
+        this.outgoingInvitations.add(invitation);
```

Comment:
since I sent you a new Invitation(), this will not work. Consult my pull request for alternative way